oak_io/src/lib.rs

```diff
@@ -138,6 +138,7 @@ impl<Init: Decodable, Command: Decodable> Decodable for InitWrapper<Init, Comman
 }
 
 #[cfg(test)]
+#[cfg(not(feature = "linear-handles"))]
```

Comment:
Is there a test that is part of CI that will run without linear handles to ensure that these tests are also executed?

Reply:
I doubt it. The error message when running with `linear-handle` is in the `drop` method. So I think all the existing tests are affected by it.